### DISCUSSION TITLE
BLOG component and moving from WP to this?

### DISCUSSION THREAD

**Discussion by janneHalttu** [givanz/Vvveb#191] [Ideas]
Is it possible to just install BLOG component and editor? Our current site is written with Svelte.
Also How much would it cost to move from WP (800 posts in 4 languages) to this? Can anyone give cost estimate for work?

**Comment by givanz** [comment]
> Is it possible to just install BLOG component and editor? 

You want to have only blog/content features and disable ecommerce? 

You can enable the `Hide Ecommerce features` plugin to disable ecommerce features https://demo.vvveb.com/admin/index.php?module=plugin/plugins#plugin-hide-ecommerce

> Our current site is written with Svelte.

Do you use WordPress in headless mode with with Svelte? Do you also need use Vvveb as headless? 

> Also How much would it cost to move from WP (800 posts in 4 languages) to this? Can anyone give cost estimate for work?

What plugin or solution do you use for Wordpress multi language? WPML, polylang, mult site install etc

There is a `Import Wordpress` plugin that can be used to import Wordpress pages and posts https://demo.vvveb.com/admin/index.php?module=plugin/plugins#plugin-import-wordpress but because WordPress does not have a standard multi language solution it does not have multi language support.

**Comment by janneHalttu** [comment]
Hi. thanks for reply.We use now WPML, but our new site has Deepl Api for ecommerce/categories translations so we would integrate blog multilanguage via API, unless you havebuilt Deepl?WP data is viewed now with Headless, via API. My site :ledstore.proJanne ***@***.***ähetetty Samsung Galaxy -älypuhelimesta.
-------- Alkuperäinen viesti --------Lähettäjä: Givan ***@***.***> Päivämäärä: 20.9.2024  21.29  (GMT+02:00) Saaja: givanz/Vvveb ***@***.***> Kopio: janneHalttu ***@***.***>, Author ***@***.***> Aihe: Re: [givanz/Vvveb] BLOG component and moving from WP to this? (Discussion #191) 

Is it possible to just install BLOG component and editor?

You want to have only blog/content features and disable ecommerce?
You can enable the Hide Ecommerce features plugin to disable ecommerce features https://demo.vvveb.com/admin/index.php?module=plugin/plugins#plugin-hide-ecommerce

Our current site is written with Svelte.

Do you use WordPress in headless mode with with Svelte? Do you also need use Vvveb as headless?

Also How much would it cost to move from WP (800 posts in 4 languages) to this? Can anyone give cost estimate for work?

What plugin or solution do you use for Wordpress multi language? WPML, polylang, mult site install etc
There is a Import Wordpress plugin that can be used to import Wordpress pages and posts https://demo.vvveb.com/admin/index.php?module=plugin/plugins#plugin-import-wordpress but because WordPress does not have a standard multi language solution it does not have multi language support.

**Reply by givanz** [reply]
Do you only need to import WPML content and use use blog/content api for your headless website or do you also need ecommerce features?

Deepl Api can be easily integrated with a plugin.
